src/DataWrangling/ecco_metadata.jl

```diff
@@ -16,6 +22,7 @@ struct ECCOMetadata{D, V}
     name  :: Symbol
     dates :: D
     version :: V
+    dir :: String
```

Comment:
what? why?

Reply:
I could also revert it back, but it is nice to know the path of the directory where the data resides, compactly in the metadata information